app/presenters/idv/welcome_presenter.rb

```diff
@@ -73,10 +74,15 @@ def bullet_points
 
     private
 
-    attr_accessor :decorated_sp_session
+    attr_accessor :decorated_sp_session, :current_user
 
     def bullet_point(bullet, text)
       OpenStruct.new(bullet: bullet, text: text)
     end
+
+    def stepping_up?
+      (current_user&.identity_verified? || current_user&.pending_profile?) &&
```

Comment:
Somewhat nit-picky, but: Despite the method name ending with `?`, this won't always produce a boolean return value, since the nil-safe operator would produce a `nil` return value if triggered. At the moment it doesn't have any impact based on how it's used in a general truthiness check.

```suggestion
    def stepping_up?
      !!(current_user&.identity_verified? || current_user&.pending_profile?) &&
```